pom.xml

```
@@ -40,6 +47,13 @@
                     <linkXRef>false</linkXRef>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
+                <configuration>
```

I'm not sure if it is necessary, but you can put version here.
```suggestion
                <artifactId>maven-war-plugin</artifactId>
                <version>3.2.3</version>
```